watertap/unit_models/tests/test_coag_floc_zo_jartest.py

```diff
@@ -0,0 +1,419 @@
+###############################################################################
+# ProteusLib Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/nawi-hub/proteuslib/"
+#
+###############################################################################
+import pytest
+from watertap.property_models.coagulation_prop_pack import CoagulationParameterBlock
+from watertap.property_models.NaCl_prop_pack import NaClParameterBlock
+from watertap.property_models.seawater_prop_pack import SeawaterParameterBlock
+from watertap.unit_models.coag_floc_zo_jartest import CoagulationFlocculationZO_JarTestModel
+from pyomo.environ import (ConcreteModel,
+                           assert_optimal_termination,
+                           value,
+                           Set,
+                           Param,
+                           Var,
+                           units as pyunits,
+                           Suffix,
+                           Constraint,
+                           SolverFactory,
+                           SolverStatus,
+                           TerminationCondition)
+from idaes.core import (FlowsheetBlock,
+                        MaterialFlowBasis,
+                        MaterialBalanceType,
+                        EnergyBalanceType)
+from idaes.core.util.exceptions import ConfigurationError
+from idaes.core.util.model_statistics import degrees_of_freedom
+from pyomo.util.check_units import assert_units_consistent
+import idaes.core.util.scaling as iscale
+from idaes.core.util.scaling import badly_scaled_var_generator
+from idaes.core.util.testing import initialization_tester
+from idaes.core.util import get_solver
+
+__author__ = "Austin Ladshaw"
+
+solver = get_solver()
+
+# -----------------------------------------------------------------------------
+# Start test class
+class TestCoagulationZOJarTest_withChemicals():
+    @pytest.fixture(scope="class")
+    def coag_obj_w_chems(self):
+        model = ConcreteModel()
+        model.fs = FlowsheetBlock(default={"dynamic": False})
+        model.fs.properties = CoagulationParameterBlock()
+        chem_dict = {'Alum':
+                        {"parameter_data":
+                            {"mw_additive": (200, pyunits.g/pyunits.mol),
```

Comment:
What chemical formula are you assuming for alum? are these dummy values?

Reply:
I know it doesn't matter for the test, but nice to have an example in the test to follow along with when creating your own model and searching for relevant inputs to get started.

Reply:
These are just dummy values. 

I will make a note in the test.